corehq/apps/es/index/analysis.py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Would it be worth adding a docstring here to note that changes to analyzers must be accompanied by a migration to apply the change to elastic?

Reply:
See b40e15fa23f